src/engine/ruleset/integrations/mysql/README.md

```diff
@@ -0,0 +1,64 @@
+# MySQL Server integration
+
+
+|   |   |
+|---|---|
+| event.module | mysql |
+| event.dataset | mysql.error, mysql.slowlog |
+
+This integration processes the following logs from [MySQL](https://dev.mysql.com/) Server:
```

Comment:
Run the generate-doc again to update this, it seems the documentation was changed without updating the readme.

Reply:
Done